### DISCUSSION TITLE
DNS with Deno Deploy?

### DISCUSSION THREAD

**Discussion by adamzerner** [denoland/deno#22401] [Q&A]
Is there a way to set DNS records with Deno Deploy without setting up a custom domain? I'm not seeing one.

The context is that I want to use [Resend](https://resend.com/overview) to send emails to users to allow them to reset their password. In order to get set up with Resend, you need to set a few DNS records. I have my app all set up with Deno Deploy, but I'm not seeing a way to set the DNS records.

<img width="1440" alt="Screenshot 2024-02-12 at 8 00 51 PM" src="https://github.com/denoland/deno/assets/3144254/0829f0dc-ba37-4035-b841-d320c69c53fc">

**Comment by nnmrts** [comment]
You'll probably have more luck getting a definitive answer here: https://github.com/denoland/deploy_feedback/issues

Anyways, you'll encounter several limitations in the future if you stick to your `.deno.dev` domain. If you get yourself a custom domain, you will be able to solve this problem for example, by changing the DNS records at the domain provider directly.

**Reply by adamzerner** [reply]
Gotcha. Just [cross posted](https://github.com/denoland/deploy_feedback/issues/622).
